src/detect/ScanI2CTwoWire.cpp

```diff
@@ -378,6 +378,11 @@ void ScanI2CTwoWire::scanPort(I2CPort port, uint8_t *address, uint8_t asize)
                 SCAN_SIMPLE_CASE(MLX90632_ADDR, MLX90632, "MLX90632 IR temp sensor found\n");
                 SCAN_SIMPLE_CASE(NAU7802_ADDR, NAU7802, "NAU7802 based scale found\n");
 
+            case SCD4X_ADDR:
```

Comment:
if you're doing another update, could use the SCAN_SIMPLE_CASE macro to keep in line with other entries here